tools/lxdclient/client.go

```diff
@@ -288,11 +325,10 @@ func checkLXDBridgeConfiguration(conf string) error {
 			}
 		} else if strings.HasPrefix(line, "LXD_BRIDGE=") {
 			name := strings.Trim(line[len("LXD_BRIDGE="):], " \"")
-			/* If we're here, we want lxdbr0 to be configured
-			 * because the default profile that juju will use says
-			 * lxdbr0. So let's fail if it doesn't.
-			 */
-			if name != "lxdbr0" {
+			// If we're here, we want the bridge to be configured because the
+			// default profile that juju will use says lxdbr0. So let's fail if
```

Comment:
This comment needs updating - it still talks about lxdbr0. (Which would retroactively justify the reformatting! ;)